Source/Krypton Components/Krypton.Toolkit/Palette Base/PaletteDefinitions.cs

```diff
@@ -2201,12 +2201,12 @@ public enum PaletteMode
         /// <summary>
         /// Specifies the visual studio dark palette theme.
         /// </summary>
-        VisualStudioDark,
+        //VisualStudioDark,
```

Comment:
Should these also be removed from the the ThemeManager as well ?

Reply:
Had a look, they don't appear in the ThemeManager